docs/self-managed/optimize-deployment/configuration/logging.md

```diff
@@ -21,21 +21,19 @@ reference as follows:
 - Optimize runtime environment:
 
 ```xml
-<logger name="org.camunda.optimize" level="info" />
+<Logger name="io.camunda.optimize" level="info" />
 ```
 
 - Optimize update:
 
 ```xml
-<logger name="org.camunda.optimize.update" level="info">
-  <appender-ref ref="UPGRADE"/>
-</logger>
+<Logger name="io.camunda.optimize.update" level="info" />
```

Comment:
I realise that this already said `update`, but should it not be `upgrade`?

Reply:
It should indeed.

When I tested it, the .upgrade has been shadowed by OPTIMIZE_LOG_LEVEL, as it has 1 package less in the path. That's why I thought it worked. Good catch!